frontend/src/api/calls/baseCalls.tsx

```diff
@@ -114,6 +121,11 @@ export function basePost(
     });
 }
 
+// Extract entity-part from a URL, for example: get /users/1 from https://localhost/api/users/1
+export function getEntityFromFullUrl(fullUrl: string): string {
+    return "/" + fullUrl.split(apiPaths.base)[1];
+}
+
```

Comment:
Currently we are using the Id's of the entity and appending them to our application paths. For consistency I would like you to keep doing the same.